shared/components/ResourceDetails/ResourceDetails.js

```diff
@@ -164,6 +168,20 @@ function Resource({
 
   const protectedResource = isProtected(resource);
 
+  const grafanaLogsAction = () => {
```

Comment:
why won't we just add it to the `headerActions` instead? I don't think it will be used in so many places that we should make it generic, let's not make this file any more chaotic.